mesheryctl/internal/cli/root/system/token.go

```diff
@@ -0,0 +1,170 @@
+package system
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var ctx string
+var tokenCmd = &cobra.Command{
+	Use:   "token",
+	Short: "Manage Meshery user tokens",
+	Long: `
+	Manipulate user tokens and their context assignments in your meshconfig`,
+	Args: cobra.MinimumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
+			return errors.New(utils.SystemError(fmt.Sprintf("invalid command: \"%s\"", args[0])))
+		}
+		return nil
+	},
+}
+
+var addTokenCmd = &cobra.Command{
+	Use:   "add",
+	Short: "Add a token to meshery config",
+	Example: `
+	mesheryctl system token add <token-name> -f <token-path>
+	mesheryctl system token add <token-name> (default path is auth.json)
+	`,
+	Args: cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		tokenName := args[0]
+		if tokenPath == "" {
+			tokenPath = "auth.json"
```

Comment:
@utkarsh-pro @navendu-pottekkat we decided on the default path being the location of `.meshery` folder, correct? @Revolyssup, was this requirement specified in the issue?

Reply:
The issue asked to set the default path as auth.json.

Reply:
Yes, the default path is ~/.meshery folder, but we eventually don't need to mention ~/.meshery in the location it can directly be auth.json we are handling it later with the help of few functions [https://github.com/meshery/meshery/blob/master/mesheryctl/internal/cli/root/constants/constants.go#L54](here)